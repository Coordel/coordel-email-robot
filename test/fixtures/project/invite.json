{
  "change": {
     "_id": "5de7629e94e88f7db2acceab7a027a75",
     "_rev": "28-cf1bd85a1ef3c84269fff41bc709e905",
     "docType": "project",
     "responsible": "2",
     "users": [
         "2",
         "1"
     ],
     "assignments": [
         {
             "username": "2",
             "role": "5de7629e94e88f7db2acceab7a050a70",
             "status": "INVITED"
         },
         {
             "username": "1",
             "role": "5de7629e94e88f7db2acceab7a03129b",
             "status": "ACCEPTED"
         }
     ],
     "name": "Test invite from DEV",
     "purpose": "Here is the purpose of the invite.\n\nWith a line break",
     "deadline": "2012-07-31",
     "status": "ACTIVE",
     "substatus": "SENT",
     "isMyDelegated": false,
     "isMyPrivate": false,
     "isTemplate": false,
     "created": "2012-07-20T14:27:18.176-04:00",
     "updated": "2012-07-22T12:51:27.197-04:00",
     "creator": "2",
     "updater": "1",
     "isNew": false,
     "history": [
         {
             "actor": {
                 "id": "2",
                 "name": "Dev Coordel",
                 "type": "PERSON"
             },
             "object": {
                 "id": "1",
                 "name": "Jeff Gorder",
                 "type": "PERSON"
             },
             "time": "2012-07-20T14:28:28.530-04:00",
             "users": [
                 "2",
                 "1"
             ],
             "target": {
                 "id": "5de7629e94e88f7db2acceab7a027a75",
                 "name": "Test invite from DEV",
                 "type": "PROJECT"
             },
             "verb": "INVITE",
             "icon": "ui-icon-arrowreturnthick-1-e"
         },
         {
             "actor": {
                 "id": "2",
                 "name": "Dev Coordel",
                 "type": "PERSON"
             },
             "object": {
                 "id": "5de7629e94e88f7db2acceab7a027a75",
                 "name": "Test invite from DEV",
                 "type": "PROJECT"
             },
             "time": "2012-07-20T14:27:24.387-04:00",
             "users": [
                 "2",
                 "1"
             ],
             "verb": "POST",
             "icon": "ui-icon-plus",
             "body": ""
         }
     ]
  },
  "username" : "1",
  "alert" : {
      "actor": {
          "id": "2",
          "name": "Dev Coordel",
          "type": "PERSON"
      },
      "object": {
          "id": "1",
          "name": "Jeff Gorder",
          "type": "PERSON"
      },
      "time": "2012-07-20T14:28:28.530-04:00",
      "users": [
          "2",
          "1"
      ],
      "target": {
          "id": "5de7629e94e88f7db2acceab7a027a75",
          "name": "Test invite from DEV",
          "type": "PROJECT"
      },
      "verb": "INVITE",
      "icon": "ui-icon-arrowreturnthick-1-e"
  }
}