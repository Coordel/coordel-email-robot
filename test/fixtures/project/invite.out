{"from":"Dev Coordel <dev@coordel.com>","to":"Jeff Gorder <jeff.gorder@coordel.com>","subject":"INVITED: Test invite from DEV","text":"Hello to myself!","html":"\n<!DOCTYPE HTML PUBLIC \"-//W3C//DTD XHTML 1.0 Transitional //EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd\"><html><head><title></title><meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\"><style type=\"text/css\">\n/* Mobile-specific Styles */\n@media only screen and (max-device-width: 480px) { \ntable[class=w0], td[class=w0] { width: 0 !important; }\ntable[class=w10], td[class=w10], img[class=w10] { width:10px !important; }\ntable[class=w15], td[class=w15], img[class=w15] { width:5px !important; }\ntable[class=w30], td[class=w30], img[class=w30] { width:10px !important; }\ntable[class=w60], td[class=w60], img[class=w60] { width:10px !important; }\ntable[class=w125], td[class=w125], img[class=w125] { width:80px !important; }\ntable[class=w130], td[class=w130], img[class=w130] { width:55px !important; }\ntable[class=w140], td[class=w140], img[class=w140] { width:90px !important; }\ntable[class=w160], td[class=w160], img[class=w160] { width:180px !important; }\ntable[class=w170], td[class=w170], img[class=w170] { width:100px !important; }\ntable[class=w180], td[class=w180], img[class=w180] { width:80px !important; }\ntable[class=w195], td[class=w195], img[class=w195] { width:80px !important; }\ntable[class=w220], td[class=w220], img[class=w220] { width:80px !important; }\ntable[class=w240], td[class=w240], img[class=w240] { width:180px !important; }\ntable[class=w255], td[class=w255], img[class=w255] { width:185px !important; }\ntable[class=w275], td[class=w275], img[class=w275] { width:135px !important; }\ntable[class=w280], td[class=w280], img[class=w280] { width:135px !important; }\ntable[class=w300], td[class=w300], img[class=w300] { width:140px !important; }\ntable[class=w325], td[class=w325], img[class=w325] { width:95px !important; }\ntable[class=w360], td[class=w360], img[class=w360] { width:140px !important; }\ntable[class=w410], td[class=w410], img[class=w410] { width:180px !important; }\ntable[class=w470], td[class=w470], img[class=w470] { width:200px !important; }\ntable[class=w580], td[class=w580], img[class=w580] { width:280px !important; }\ntable[class=w640], td[class=w640], img[class=w640] { width:300px !important; }\ntable[class*=hide], td[class*=hide], img[class*=hide], p[class*=hide], span[class*=hide] { display:none !important; }\ntable[class=h0], td[class=h0] { height: 0 !important; }\np[class=footer-content-left] { text-align: center !important; }\n#headline p { font-size: 30px !important; }\n.article-content, #left-sidebar{ -webkit-text-size-adjust: 90% !important; -ms-text-size-adjust: 90% !important; }\n.header-content, .footer-content-left {-webkit-text-size-adjust: 80% !important; -ms-text-size-adjust: 80% !important;}\n } \n/* Client-specific Styles */\n#outlook a { padding: 0; }\t/* Force Outlook to provide a \"view in browser\" button. */\nbody { width: 100% !important; }\n.ReadMsgBody { width: 100%; }\n.ExternalClass { width: 100%; display:block !important; } /* Force Hotmail to display emails at full width */\n/* Reset Styles */\n/* Add 100px so mobile switch bar doesn't cover street address. */\nbody { background-color: #dedede; margin: 0; padding: 0; }\nimg { height: auto; line-height: 100%; outline: none; text-decoration: none; display: block;}\nbr, strong br, b br, em br, i br { line-height:100%; }\nh1, h2, h3, h4, h5, h6 { line-height: 100% !important; -webkit-font-smoothing: antialiased; }\nh1 a, h2 a, h3 a, h4 a, h5 a, h6 a { color: blue !important; }\nh1 a:active, h2 a:active,  h3 a:active, h4 a:active, h5 a:active, h6 a:active {\tcolor: red !important; }\n/* Preferably not the same color as the normal header link color.  There is limited support for psuedo classes in email clients, this was added just for good measure. */\nh1 a:visited, h2 a:visited,  h3 a:visited, h4 a:visited, h5 a:visited, h6 a:visited { color: purple !important; }\n/* Preferably not the same color as the normal header link color. There is limited support for psuedo classes in email clients, this was added just for good measure. */  \ntable td, table tr { border-collapse: collapse; }\n.yshortcuts, .yshortcuts a, .yshortcuts a:link,.yshortcuts a:visited, .yshortcuts a:hover, .yshortcuts a span {\ncolor: black; text-decoration: none !important; border-bottom: none !important; background: none !important;\n}\t/* Body text color for the New Yahoo.  This example sets the font of Yahoo's Shortcuts to black. */\n/* This most probably won't work in all email clients. Don't include <code _tmplitem=\"431\" > blocks in email. */\ncode {\n  white-space: normal;\n  word-break: break-all;\n}\n#background-table { background-color: #dedede; }\n/* Webkit Elements */\n#top-bar { border-radius:6px 6px 0px 0px; -moz-border-radius: 6px 6px 0px 0px; -webkit-border-radius:6px 6px 0px 0px; -webkit-font-smoothing: antialiased; background-color: #00629B; color: #ededed; }\n#top-bar a { font-weight: bold; color: #ffffff; text-decoration: none;}\n#footer { border-radius:0px 0px 6px 6px; -moz-border-radius: 0px 0px 6px 6px; -webkit-border-radius:0px 0px 6px 6px; -webkit-font-smoothing: antialiased; }\n/* Fonts and Content */\nbody, td { font-family: 'Helvetica Neue', Arial, Helvetica, Geneva, sans-serif; }\n.header-content, .footer-content-left, .footer-content-right { -webkit-text-size-adjust: none; -ms-text-size-adjust: none; }\n/* Prevent Webkit and Windows Mobile platforms from changing default font sizes on header and footer. */\n.header-content { font-size: 12px; color: #ededed; }\n.header-content a { font-weight: bold; color: #ffffff; text-decoration: none; }\n#headline p { color: #444444; font-family: 'Helvetica Neue', Arial, Helvetica, Geneva, sans-serif; font-size: 36px; text-align: center; margin-top:0px; margin-bottom:30px; }\n#headline p a { color: #444444; text-decoration: none; }\n.article-title { font-size: 16px; line-height:24px; color: #00629B; font-weight:bold; margin-top:20px; margin-bottom:18px; font-family: 'Helvetica Neue', Arial, Helvetica, Geneva, sans-serif; }\n.article-title a { color: #00629B; text-decoration: none; }\n.article-title.with-meta {margin-bottom: 0;}\n.article-meta { font-size: 13px; line-height: 20px; color: #ccc; font-weight: bold; margin-top: 0;}\n.article-content { font-size: 13px; line-height: 18px; color: #333333; margin-top: 0px; margin-bottom: 18px; font-family: 'Helvetica Neue', Arial, Helvetica, Geneva, sans-serif; }\n.article-content-large { font-size: 16px;}\n.indented {margin-left: 20px; margin-right: 20px;}\n.article-content a { color: #642F6C; font-weight:bold; text-decoration:none; }\n.article-content img { max-width: 100% }\n.article-content ol, .article-content ul { margin-top:0px; margin-bottom:18px; margin-left:19px; padding:0; }\n.article-content li { font-size: 13px; line-height: 18px; color: #333333; }\n.article-content li a { color: #642F6C; text-decoration:underline; }\n.article-content p {margin-bottom: 15px;}\n.footer-content-left { font-size: 12px; line-height: 15px; color: #ededed; margin-top: 0px; margin-bottom: 15px; }\n.footer-content-left a { color: #ffffff; font-weight: bold; text-decoration: none; }\n.footer-content-right { font-size: 11px; line-height: 16px; color: #ededed; margin-top: 0px; margin-bottom: 15px; }\n.footer-content-right a { color: #ffffff; font-weight: bold; text-decoration: none; }\n#footer { background-color: #00629B; color: #ededed; }\n#footer a { color: #ffffff; text-decoration: none; font-weight: bold; }\n#permission-reminder { white-space: normal; }\n#street-address { color: #97D700; white-space: normal; }\n.button {\n\tbackground: #d8362b; /* Old browsers */\n/* IE9 SVG, needs conditional override of 'filter' to 'none' */\nbackground: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/Pgo8c3ZnIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjEwMCUiIGhlaWdodD0iMTAwJSIgdmlld0JveD0iMCAwIDEgMSIgcHJlc2VydmVBc3BlY3RSYXRpbz0ibm9uZSI+CiAgPGxpbmVhckdyYWRpZW50IGlkPSJncmFkLXVjZ2ctZ2VuZXJhdGVkIiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIgeDE9IjAlIiB5MT0iMCUiIHgyPSIwJSIgeTI9IjEwMCUiPgogICAgPHN0b3Agb2Zmc2V0PSIwJSIgc3RvcC1jb2xvcj0iI2Q4MzYyYiIgc3RvcC1vcGFjaXR5PSIxIi8+CiAgICA8c3RvcCBvZmZzZXQ9IjQ1JSIgc3RvcC1jb2xvcj0iI2Q2MjIxNSIgc3RvcC1vcGFjaXR5PSIxIi8+CiAgICA8c3RvcCBvZmZzZXQ9IjEwMCUiIHN0b3AtY29sb3I9IiNkYTI5MWMiIHN0b3Atb3BhY2l0eT0iMSIvPgogIDwvbGluZWFyR3JhZGllbnQ+CiAgPHJlY3QgeD0iMCIgeT0iMCIgd2lkdGg9IjEiIGhlaWdodD0iMSIgZmlsbD0idXJsKCNncmFkLXVjZ2ctZ2VuZXJhdGVkKSIgLz4KPC9zdmc+);\nbackground: -moz-linear-gradient(top,  #d8362b 0%, #d62215 45%, #da291c 100%); /* FF3.6+ */\nbackground: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#d8362b), color-stop(45%,#d62215), color-stop(100%,#da291c)); /* Chrome,Safari4+ */\nbackground: -webkit-linear-gradient(top,  #d8362b 0%,#d62215 45%,#da291c 100%); /* Chrome10+,Safari5.1+ */\nbackground: -o-linear-gradient(top,  #d8362b 0%,#d62215 45%,#da291c 100%); /* Opera 11.10+ */\nbackground: -ms-linear-gradient(top,  #d8362b 0%,#d62215 45%,#da291c 100%); /* IE10+ */\nbackground: linear-gradient(to bottom,  #d8362b 0%,#d62215 45%,#da291c 100%); /* W3C */\nfilter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#d8362b', endColorstr='#da291c',GradientType=0 ); /* IE6-8 */\n border-radius:6px 6px 6px 6px; -moz-border-radius: 6px 6px 6px 6px; -webkit-border-radius:6px 6px 6px 6px; -webkit-font-smoothing: antialiased;\n color: #ffffff;\n\n padding: 4px;\n width: 200px;\n text-align:center;\n font-size:14px\n}\n.action-required {\n  color: #da291c;\n}\n.button a {\n\tcolor: #ffffff;\n\ttext-decoration: none;\n}\n.button:hover{\n\tcursor:pointer;\n}\n.hidden {\n  display: none;\n}\n\n</style>\n<!--[if gte mso 9]>\n<style _tmplitem=\"431\" >\n.article-content ol, .article-content ul {\n   margin: 0 0 0 24px;\n   padding: 0;\n   list-style-position: inside;\n}\n</style>\n<![endif]-->\n\n<!--[if gte IE 9]>\n  <style type=\"text/css\">\n    .gradient {\n       filter: none;\n    }\n  </style>\n<![endif]-->\n\n</head>\n<body>\n\t<table width=\"100%\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\" id=\"background-table\">\n\t\t<tbody>\n\t\t<tr>\n\t\t\t<td align=\"center\" bgcolor=\"#dedede\">\n        \t\t<table class=\"w640\" style=\"margin:0 10px;\" width=\"640\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\">\n            \t\t<tbody>\n            \t\t<tr>\n            \t\t\t<td class=\"w640\" width=\"640\" height=\"20\"></td>\n            \t\t</tr>\n            \t\t<tr>\n            \t\t\t<td class=\"w640\" width=\"640\" style=\"padding-bottom: 10px;\">\n            \t\t\t\t<img src=\"http://dev.coordel.com:8090/images/coordel-logo.png\" height=\"64\" width=\"247\">\n            \t\t\t</td>\n            \t\t</tr>\n            \t\t<tr>\n                \t\t<td class=\"w640\" width=\"640\">\n                        \t<table id=\"top-bar\" class=\"w640\" width=\"640\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\" bgcolor=\"#ffffff\">\n\t                        \t<tbody>\n\t                        \t<tr>\n\t                        \t\t<td class=\"w15\" width=\"15\"></td>\n\t                        \t\t<td class=\"w325\" width=\"350\" valign=\"middle\" align=\"left\">\n\t\t                        \t\t<table class=\"w325\" width=\"350\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\">\n\t\t\t                        \t\t<tbody>\n\t\t\t                        \t\t<tr>\n\t\t\t                        \t\t\t<td class=\"w325\" width=\"350\" height=\"8\"></td>\n\t\t\t                        \t\t</tr>\n\t\t\t                        \t\t</tbody>\n\t\t\t                        \t</table>\n            \n\t\t\t                        \t<table class=\"w325\" width=\"350\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\">\n\t\t\t\t                        \t<tbody>\n\t\t\t\t                        \t<tr>\n\t\t\t\t                        \t\t<td class=\"w325\" width=\"350\" height=\"8\"></td>\n\t\t\t\t                        \t</tr>\n\t\t\t\t                        \t</tbody>\n\t\t\t\t                        </table>\n\t\t\t\t                    </td>\n\t\t\t\t                    <td class=\"w30\" width=\"30\"></td>\n\t\t\t\t                    <td class=\"w255\" width=\"255\" valign=\"middle\" align=\"right\">\n\t\t\t\t\t                    <table class=\"w255\" width=\"255\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\">\n\t\t\t\t\t\t                    <tbody>\n\t\t\t\t\t\t                    <tr><td class=\"w255\" width=\"255\" height=\"8\"></td></tr>\n\t\t\t\t\t\t                    </tbody>\n\t\t\t\t\t\t                </table>\n\t\t\t\t\t\t                <table cellpadding=\"0\" cellspacing=\"0\" border=\"0\">\n\t\t\t\t\t\t\t                <tbody>\n\t\t\t\t\t\t\t                <tr></tr>\n\t\t\t\t\t\t\t                </tbody>\n\t\t\t\t\t\t\t            </table>\n\t\t\t\t\t\t\t            <table class=\"w255\" width=\"255\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\">\n\t\t\t\t\t\t\t\t            <tbody>\n\t\t\t\t\t\t\t\t            <tr><td class=\"w255\" width=\"255\" height=\"8\"></td></tr>\n\t\t\t\t\t\t\t\t            </tbody>\n\t\t\t\t\t\t\t\t        </table>\n\t\t\t\t\t\t\t\t    </td>\n\t\t\t\t\t\t\t\t    <td class=\"w15\" width=\"15\"></td>\n\t\t\t\t\t\t\t\t</tr>\n\t\t\t\t\t\t\t\t</tbody>\n\t\t\t\t\t\t\t</table>\n\t\t\t\t\t\t</td>\n\t\t\t\t\t</tr>\n\t\t\t\t\t<tr>\n\t\t\t\t\t\t<td class=\"w640\" width=\"640\" height=\"30\" bgcolor=\"#ffffff\"></td>\n\t\t\t\t\t</tr>\n\t\t\t\t\t<tr id=\"simple-content-row\">\n\t\t\t\t\t\t<td class=\"w640\" width=\"640\" bgcolor=\"#ffffff\">\n\t\t\t\t\t\t\t<table class=\"w640\" width=\"640\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\">\n\t\t\t\t\t\t\t\t<tbody>\n\t\t\t\t\t\t\t\t<tr>\n\t\t\t\t\t\t\t\t\t<td class=\"w30\" width=\"30\"></td>\n\t\t\t\t\t\t\t\t\t<td class=\"w580\" width=\"580\">\n\t\t\t\t\t                    <layout label=\"Main message\">\n\t\t\t\t\t                        <table class=\"w580\" width=\"580\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\">\n\t\t\t\t\t                        \t<tbody>\n\t\t\t\t\t                            <tr>\n\t\t\t\t\t                                <td class=\"w580\" width=\"580\"> \n\t\t\t\t\t                                    <div id=\"mainMessage\" align=\"left\" class=\"article-content-large\"><layout label=\"Project Invite Activity\">\n  <table class=\"w580\" width=\"580\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\">\n    <tbody>\n      <tr>\n        <td class=\"w580\" width=\"580\">\n        <p align=\"left\" class=\"article-content\" id=\"time\">Friday, July 20th, 2012, 2:28:28 PM</p>\n\t      <p align=\"left\" class=\"article-content-large\"><span id=\"sender\">Dev Coordel</span> <strong>invited</strong> you to work together on a new Coord. Coords coordinate people and tasks related by a common purpose into a highly productive process. Here are the Coord's details:</p>\n\t      <div class=\"indented\" class=\"article-content-large\">\n\t\t      <p align=\"left\"><strong>Name:</strong> <span id=\"coord\">Test invite from DEV</span></p>\n\t\t      <p align=\"left\" class=\"purpose_entry\"><strong>Purpose:</strong> <span id=\"purpose\">Here is the purpose of the invite.<br><br>With a line break</span></p>\n\t\t      <p align=\"left\" class=\"deadline_entry\"><strong>Deadline:</strong> <span id=\"deadline\">Monday, July 30, 2012</span></p>\n\t      </div>\n        </td>\n      </tr>\n      <tr>\n        <td class=\"w580\" width=\"580\" height=\"15\"></td>\n      </tr>\n      <tr>\n        <td class=\"w580\" width=\"580\">\n          <div align=\"left\" class=\"article-content-large\">\n            <strong class=\"action-required\">Action Required:</strong> Sign on to Coordel to <em>accept</em>, <em>decline</em>, or <em>propose changes</em> to the invitation.\n          </div>\n        </td>\n      </tr>\n      <tr>\n        <td class=\"w580\" width=\"580\" height=\"10\"></td>\n      </tr>\n    </tbody>\n  </table>\n</layout></div>\n\t\t\t\t\t                                </td>\n\t\t\t\t\t                            </tr>\n\t\t\t\t\t                            <tr>\n\t\t\t\t\t                            \t<td class=\"w580\" width=\"580\" height=\"10\"></td>\n\t\t\t\t\t                            </tr>\n\t\t\t\t\t                            </tbody>\n\t\t\t\t\t                        </table>\n\t\t\t\t\t                    </layout>\n                \n\t\t\t\t\t                    <layout label=\"Invited user message\">\n\t\t\t\t\t                        <table class=\"w580\" width=\"580\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\">\n\t\t\t\t\t                            <tbody>\n\t\t\t\t\t                            <tr id=\"button\"><tr>\n  <td class=\"w580\" width=\"580\" align=\"center\">\n    <div class=\"button gradient\">\n      <a class=\"button-link\" href=\"http://dev.coordel.com:8080/login\">Login</a>\n    </div>\n  </td>\n</tr>\n<tr><td class=\"w580\" width=\"580\" height=\"10\"></td></tr>\n\n\n\n\n</tr>\n\t\t\t\t\t                            <tr><td class=\"w580\" width=\"580\" height=\"10\"></td></tr>\n\t\t\t\t\t                            </tbody>\n\t\t\t\t\t                         </table>\n\t\t\t\t\t                    </layout>\n                \n\t\t\t\t\t               \n\t\t\t\t\t\t\t\t\t</td>\n\t\t\t\t\t\t\t\t</tr>       \n\t\t\t\t\t\t        </tbody>\n\t\t\t\t\t\t    </table>\n\t\t\t\t\t\t</td>\n\t\t\t\t\t</tr>\n\t\t\t\t\t<tr>\n\t\t\t\t\t\t<td class=\"w640\" width=\"640\" height=\"15\" bgcolor=\"#ffffff\"></td>\n\t\t\t\t\t</tr>\n    \n\t\t\t\t\t<tr>\n\t\t\t\t\t\t<td class=\"w640\" width=\"640\">\n\t\t\t\t\t\t\t<table id=\"footer\" class=\"w640\" width=\"640\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\" bgcolor=\"#00629B\">\n\t\t\t\t\t\t\t\t<tbody>\n\t\t\t\t\t\t\t\t<tr>\n\t\t\t\t\t\t\t\t\t<td class=\"w30\" width=\"30\"></td><td class=\"w580 h0\" width=\"360\" height=\"30\"></td><td class=\"w0\" width=\"60\"></td><td class=\"w0\" width=\"160\"></td>\n\t\t\t\t\t\t\t\t\t<td class=\"w30\" width=\"30\"></td>\n\t\t\t\t\t\t\t\t</tr>\n\t\t\t\t\t\t\t\t<tr>\n\t\t\t\t\t\t\t\t\t<td class=\"w30\" width=\"30\"></td>\n\t\t\t\t\t\t\t\t\t<td class=\"w580\" width=\"360\" valign=\"top\">\n\t\t\t\t\t\t\t\t\t\t<span class=\"hide\"><p id=\"permission-reminder\" align=\"left\" class=\"footer-content-left\"><span id=\"reason\">You're receiving this because you ...</span></p></span>\n\t\t\t\t\t\t\t\t\t\t<p align=\"left\" class=\"footer-content-left\"><a href=\"http://coordel.updatemyprofile.com/j-l-2AD73FFF-l-t\">Edit your subscription</a> | <a href=\"http://coordel.createsend1.com/t/j-u-l-l-i/\">Unsubscribe</a></p>\n\t\t\t\t\t\t\t\t\t</td>\n\t\t\t\t\t\t\t\t\t<td class=\"hide w0\" width=\"60\"></td>\n\t\t\t\t\t\t\t\t\t<td class=\"hide w0\" width=\"160\" valign=\"top\">\n\t\t\t\t\t\t\t\t\t\t<p id=\"street-address\" align=\"right\" class=\"footer-content-right\"><span>Coordel - love what you do</span></p>\n\t\t\t\t\t\t\t\t\t</td>\n\t\t\t\t\t\t\t\t\t<td class=\"w30\" width=\"30\"></td>\n\t\t\t\t\t\t\t\t</tr>\n\t\t\t\t\t\t\t\t<tr>\n\t\t\t\t\t\t\t\t\t<td class=\"w30\" width=\"30\"></td><td class=\"w580 h0\" width=\"360\" height=\"15\"></td>\n\t\t\t\t\t\t\t\t\t<td class=\"w0\" width=\"60\"></td><td class=\"w0\" width=\"160\"></td>\n\t\t\t\t\t\t\t\t\t<td class=\"w30\" width=\"30\"></td>\n\t\t\t\t\t\t\t\t</tr>\n\t\t\t\t\t\t\t\t</tbody>\n\t\t\t\t\t\t\t</table>\n\t\t\t\t\t\t</td>\n\t\t\t\t\t</tr>\n\t\t\t\t\t<tr><td class=\"w640\" width=\"640\" height=\"60\"></td></tr>\n\t\t\t\t\t</tbody>\n\t\t\t\t</table>\n\t\t\t</td>\n\t\t</tr>\n\t\t</tbody>\n\t</table>\n</body>\n</html>\n"}